_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -945,6 +945,48 @@ mfxStatus VideoDECODEVP9_HW::PrepareInternalSurface(UMC::FrameMemID &mid, mfxFra
     return MFX_ERR_NONE;
 }
 
+
+static mfxStatus CheckFrameInfo(mfxFrameInfo &currInfo, mfxFrameInfo &info)
+{
+    mfxStatus sts = CheckFrameInfoCommon(&info, MFX_CODEC_VP9);
+
+    if (sts != MFX_ERR_NONE)
```

Comment:
I think you can use macro like ```MFX_CHECK``` here. It is preferable, since they support error tracing. Same for all checks below

Reply:
Tests failed with this macro, so it has little different behaviour

Reply:
What do you mean different? If you write:
```
MFX_CHECK_STS(sts);
```
or
```
MFX_CHECK(sts == MFX_ERR_NONE, sts);
```
it will have exactly the same behavior but will allow you to trace the error origin

Reply:
Yes, it works, updated